qiskit_aer/backends/aer_compiler.py

```diff
@@ -823,6 +823,16 @@ def _assemble_op(
     }:
         aer_circ.gate(name, qubits, params, [], conditional_reg, aer_cond_expr,
                       label if label else name)
+    elif "_o0" in name:
+        orig_name = name.replace("_o0", "")
+        qubits_0 = [qubits[0]]
+        aer_circ.gate("x", qubits_0, params, [], conditional_reg, aer_cond_expr,
+                      label if label else "x")
+        aer_circ.gate(orig_name, qubits, params, [], conditional_reg, aer_cond_expr,
```

Comment:
I think we need to check `orig_name` is in supported gates in `AerCircuit`. In addition, `mc*` will use more variation of `ctrl_state` in addition to `_o0` (such as `_o0101`).  

I think `transpiler` is good to decompose gates with basis gates. Or we say "gate with `ctrl_state` is not supported".